django/cantusdb_project/main_app/views/source.py

```diff
@@ -268,7 +308,7 @@ def form_valid(self, form):
         source = form.save()
 
         for old_editor in old_current_editors:
-            old_editor.sources_user_can_edit.remove(source)
+            old_editor.sources_3.remove(source)
```

Comment:
what is `sources_3`?

Reply:
oh! I don't recall making this change - something must have happened while sorting out the merge conflict. Thanks for catching this!